docs/notebooks/lda_training_tips.ipynb

```diff
@@ -0,0 +1,726 @@
+      "   (0.0026645898310183875, 'scale'),\n",
+      "   (0.0026553222554916082, 'word'),\n",
+      "   (0.0026489429534993759, 'nearest')],\n",
+      "  -199.50365509313696),\n",
+      " ([(0.014619762720062554, 'circuit'),\n",
+      "   (0.013163620299223806, 'neuron'),\n",
+      "   (0.011173418735156853, 'chip'),\n",
+      "   (0.010838272171877458, 'analog'),\n",
+      "   (0.0096353380726272708, 'signal'),\n",
+      "   (0.0080391211812232306, 'voltage'),\n",
+      "   (0.0055581577899570835, 'channel'),\n",
+      "   (0.0054775778900036506, 'noise'),\n",
+      "   (0.0052134436268982988, 'vlsi'),\n",
+      "   (0.0048225583229723852, 'bit'),\n",
+      "   (0.0044179271415801871, 'implementation'),\n",
+      "   (0.0042542648528188362, 'cell'),\n",
+      "   (0.0039531536498857694, 'frequency'),\n",
+      "   (0.0038684482611307152, 'pulse'),\n",
+      "   (0.0036454420814875429, 'synapse'),\n",
+      "   (0.0036188976174382961, 'threshold'),\n",
+      "   (0.0032522992281379531, 'gate'),\n",
+      "   (0.0032069531663885243, 'fig'),\n",
+      "   (0.0031674367038267578, 'filter'),\n",
+      "   (0.0031642918140801749, 'digital')],\n",
+      "  -203.62039767672215),\n",
+      " ([(0.016495397244929412, 'speech'),\n",
+      "   (0.014224555986108658, 'word'),\n",
+      "   (0.014198504253159445, 'recognition'),\n",
+      "   (0.0088937679870855438, 'layer'),\n",
+      "   (0.0072520103636913641, 'classifier'),\n",
+      "   (0.0067167934159034458, 'speaker'),\n",
+      "   (0.0063553921838090102, 'context'),\n",
+      "   (0.0062578159284403748, 'class'),\n",
+      "   (0.0059853201329598928, 'hidden'),\n",
+      "   (0.0056887097559640606, 'hmm'),\n",
+      "   (0.0055328441630430568, 'classification'),\n",
+      "   (0.0049471802300725225, 'trained'),\n",
+      "   (0.0047306434471966301, 'frame'),\n",
+      "   (0.0044173332526889391, 'phoneme'),\n",
+      "   (0.0043987903208920357, 'mlp'),\n",
+      "   (0.0041175501101671941, 'net'),\n",
+      "   (0.0038161702513077969, 'acoustic'),\n",
+      "   (0.0037067168884100422, 'speech_recognition'),\n",
+      "   (0.0035800864004930716, 'rbf'),\n",
+      "   (0.0035026237420255455, 'architecture')],\n",
+      "  -220.48682168542942)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "top_topics = model.top_topics(corpus, num_words=20)\n",
+    "\n",
+    "# Average topic coherence is the sum of topic coherences of all topics, divided by the number of topics.\n",
+    "avg_topic_coherence = sum([t[1] for t in top_topics]) / num_topics\n",
+    "print('Average topic coherence: %.4f.' % avg_topic_coherence)\n",
+    "\n",
+    "from pprint import pprint\n",
+    "pprint(top_topics)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[pyLDAvis](https://pyldavis.readthedocs.io/en/latest/index.html) can be fun and useful. Include the code below in your notebook to visualize your topics with pyLDAvis.\n",
```

Comment:
I removed the text about it. Come to think of it, since it's mentioned in other RaRe blogs, there isn't much need for it in this one.